pytube/reasons.py

```diff
@@ -0,0 +1,7 @@
+# This dictionary can be used for mapping playability issues' reasons
+# to players to use to bypass said reason.
+PLAYER_FOR_REASON = {
+  'Playback on other websites has been disabled by the video owner': 'ANDROID',
+  'This video is not available': 'ANDROID',
+  'This video is unavailable': 'ANDROID',
+}
```

Comment:
This module's name and this variable name are not very clear. I don't think it needs its own separate module.
This could be better documented with nested key/value pairs, ie 
```python
FALLBACK_CLIENTS = [
    {
        'error_message': 'Playback on other websites has been disabled by the video owner',
        'client': 'ANDROID'
    },
    ...
]
```

Reply:
Then again I don't think this should be handled here.